internal/zinc-persist/src/main/scala/sbt/internal/inc/TextAnalysisFormat.scala

```diff
@@ -266,6 +279,28 @@ object TextAnalysisFormat {
     }
   }
 
+  private[this] object FullAPIsF {
```

Comment:
I think we should store there only companions - we don't want to store compilations and name hashes twice (especially name hashes are heavy)